packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -230,6 +237,10 @@ export type CallAdapterClientState = {
     alternateCallerId?: string;
 };
 
+// @public
+export interface CallAdapterCommon extends AdapterState<CallAdapterState>, Disposable, CallAdapterCallManagement, CallAdapterDeviceManagement, CallAdapterSubscribers {
```

Comment:
Can we avoid exporting the common interface and just publicly export `CallAdapter` and `TeamsCallAdapter` and use `CallAdapter | TeamsCallAdapter` where this is used? (in the public interface)

Reply:
Can't. `CallComposite` now only needs the `CallAdapterCommon` interface, not `CallAdapter`. This is because the composite only makes use of the common functionality.